### PR TITLE
GEOPY-522 - Create a driver/param for iso-surface

### PR DESCRIPTION
**Pull request by jamiefb**
**GEOPY-522 - Create a driver/param for iso-surface**

### CODE REVIEW

**Review by benk-mira — COMMENTED** [review]
Nice work Jamie - A great contribution!  Just a couple comments to address, but I think we should loop in @domfournier since they're for code that was in the original application.

File: geoapps/iso_surfaces/params.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    @property
+    def contours(self) -> str | None:
+        """
+        String defining sets of contours.
+        Contours can be defined over an interval `50:200:10` and/or at a fix value `215`.
+        Any combination of the above can be used:
+        50:200:10, 215 => Contours between values 50 and 200 every 10, with a contour at 215.
+        """
+        return self._contours
```
[the remark this answers is not in the document]
Comment (domfournier) [reply geoapps/iso_surfaces/params.py:76]: I agree, but that's going to be a re-tooling of the existing code. Let's do it on a separate issue.

[the remark this answers is not in the document]
Comment (benk-mira) [reply geoapps/iso_surfaces/params.py:76]: yep, that's fine

[the remark this answers is not in the document]
Comment (domfournier) [reply geoapps/iso_surfaces/params.py:76]: To be addressed on GEOPY-530

File: geoapps/iso_surfaces/constants.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+from uuid import UUID
+
+from geoh5py.ui_json.constants import default_ui_json as base_ui_json
+
+defaults = {
+    "title": "Create Iso Surfaces",
+    "geoh5": None,
+    "objects": None,
+    "data": None,
+    "contours": "0.005: 0.02: 0.005, 0.0025",
```
[the remark this answers is not in the document]
Comment (domfournier) [reply geoapps/iso_surfaces/constants.py:18]: To be addressed on GEOPY-530

**Review by domfournier — DISMISSED** [review]

**Review by domfournier — APPROVED** [review]

**Pull request merged**